pkgs/development/libraries/agda/generic/default.nix

```diff
@@ -0,0 +1,37 @@
+{ lib, mkDerivation, fetchFromGitHub, standard-library }:
+
+mkDerivation rec {
+  pname = "generic";
+  version = "0.1";
+
+  src = fetchFromGitHub {
+    repo = "Generic";
+    owner = "effectfully";
+    rev = "${version}";
+    sha256 = "04a878g0af0pkaxasz4x6cqqh5y0js5d9pv7vf6dd0dq3w0qn1ks";
+  };
+
+  buildInputs = [
+    standard-library
+  ];
+
+  # # everythingFile = "src/Generic/Main.agda";
```

Comment:
I guess you want to remove these comments

Reply:
yes, just wanted to show the things I tried